core/math/disjoint_set.h

```diff
@@ -0,0 +1,155 @@
+/*************************************************************************/
+/*  disjoint_set.h                                                       */
```

Comment:
This file is only used in the GLTF module, does it belong in core?

Reply:
It is a mathematics algorithm, so belongs in math. @marstaik Thoughts?

Reply:
Alright, then we should keep it. It was worth asking the question though :)